java/test/jmri/jmrit/operations/locations/schedules/tools/Bundle.java

```diff
@@ -0,0 +1,99 @@
+package jmri.jmrit.operations.locations.schedules.tools;
```

Comment:
What's the purpose of a Bundle class in the test suite? I don't think we have that in other places.

Reply:
That's a mistake.  Should be a test bundle.